libnemo-private/nemo-action-manager.c

```diff
@@ -413,6 +413,24 @@ nemo_action_manager_list_actions (NemoActionManager *action_manager)
     return action_manager->action_list_dirty ? NULL : action_manager->actions;
 }
 
+gchar *
+nemo_action_manager_get_system_directory_path (const gchar *data_dir)
+{
+    const gchar *nemo_path, *target;
+
+	nemo_path = g_build_filename (data_dir, "nemo", NULL);
+
+    // For symbolic links, we try to figure out its actual path
+    // to prevent possible duplicate right-click menu items.
+	target = g_file_read_link (nemo_path, NULL);
+
+	if (target) {
+		return g_build_filename (target, "actions", NULL);
+	} else {
+		return g_build_filename (nemo_path, "actions", NULL);
+	}
+}
+
```

Comment:
Can you make sure there are only spaces used here (currently you have a mix of tabs and spaces, which messes up indentation here).